core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -138,6 +139,41 @@ public void testNullStatusOnKraftCommandAlterClientMetrics() {
         assertEquals("Completed updating config for client-metric cm.", message);
     }
 
+    @ClusterTest
+    public void testAddConfigKeyValuesUsingCommand() throws Exception {
+        String topicName = "test-topic";
+
+        try (Admin client = cluster.createAdminClient()) {
+            NewTopic newTopic = new NewTopic(topicName, 1, (short) 1);
+            client.createTopics(Collections.singleton(newTopic)).all().get();
+        }
+
+        String configKey = "cleanup.policy";
+        List<String> configValues = List.of("delete", "compact");
+        String configValueStr = String.join(",", configValues);
+
+        Stream<String> command = Stream.concat(quorumArgs(), Stream.of(
+                "--entity-type", "topics",
+                "--entity-name", topicName,
```

Comment:
To follow the coding style in this file, like in `testNullStatusOnKraftCommandAlterClientMetrics`, it’s better to explicitly list constant strings instead of storing them in variables.

Reply:
ok.done.